bluesky/run_engine.py

```diff
@@ -1849,6 +1850,27 @@ async def _read(self, msg):
 
         return ret
 
+    async def _locate(self, msg: Msg):
+        """
+        Locate some Movables and return their locations.
+
+        Expected message object is:
+
+            Msg('locate', obj1, ..., objn)
+
+        If a single obj is passed, obj.locate() is returned. If multiple objs
+        are passed, obj.locate() is called in parallel for all objs and a list
+        of the results returned.
+
+        """
+        objs = [check_supports(obj, Locatable) for obj in (msg.obj,) + msg.args]
+        # actually _locate_ the objects
+        coros = [maybe_await(obj.locate()) for obj in objs]
+        if len(coros) == 1:
```

Comment:
I have a weak preference to always return a list even if it is length 1.  I think it is slightly more verbose if you always just use 1, but much nicer if you want to write generic code that is 0 or more.